src/picterra/client.py

```diff
@@ -306,6 +306,58 @@ def list_rasters(self, folder_id=None):
         params = {'folder': folder_id} if folder_id else {}
         return self._paginate_through_list('rasters', params)
 
+    def edit_raster(
+        self, raster_id: str,
+        name: Optional[str]=None, folder_id: Optional[str]=None,
+        captured_at: Optional[str]=None, identity_key:Optional[str]=None,
+        multispectral_band_specification: Optional[dict] = None
+    ):
+        """
+        Edits an already existing raster.
+
+        Args:
+            name: New human-readable name for this raster
+            folder_id: Id of the new folder for this raster (move is in another project)
+            captured_at: new ISO-8601 date and time at which this
+                raster was captured, YYYY-MM-DDThh:mm[:ss[.uuuuuu]][+HH:MM|-HH:MM|Z];
+                e.g. "2020-01-01T12:34:56.789Z"
+            identity_key: New personal identifier for this raster.
+            multispectral_band_specification: If True, the raster is in multispectral mode and can have an associated band specification
+
+        Returns:
+            raster_id (str): The id of the uploaded raster
```

Comment:
The comment here should be "The id of the *edited* raster"

Reply:
fixed